dive_core/common/emulate_pm4.h

```diff
@@ -191,7 +191,7 @@ class EmulatePM4
     // Keep all emulation state together
     struct EmulateState
     {
-        static const uint32_t kMaxPendingIbs = 100;
```

Comment:
Can you give context on this?
The reason it's 100 is because I didn't think set_draw_state ever exceeded 100 count. Are you saying that's possible?

If we're exceeding this # for a non-set-draw-state, then we could be dealing with a bug.

Reply:
replied in the chatroom, we can continue the discussion there. Benson has more info on the capture.